### PR TITLE
Feature-add-Image-tag

### PR DESCRIPTION
**Pull request by yahyagulshan**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -17,17 +20,27 @@ pipeline {
                    sh 'docker login -u yahya4246 -p ${dockerhubpwd}'
 
 }
-                   sh 'docker push yahya4246/jenkins-image'
+                   sh 'docker push yahya4246/jenkins-image:${IMAGE_TAG}'
```
Comment (abaidgulshan) [inline Jenkinsfile]: Add Docker image as tag as well in enviroment 
DOCKER_REPO = "yahya4246/jenkins-image"

File: Jenkinsfile [hunk 2]
```diff
@@ -17,17 +20,27 @@ pipeline {
...
+        }
+
         stage('Deploy App on k8s') {
       steps {
+  
         withCredentials([
             string(credentialsId: 'my_kubernetes', variable: 'api_token')
             ]) {
-             sh 'kubectl --token $api_token --server http://127.0.0.1:35153/  --insecure-skip-tls-verify=true apply -f deployment.yaml '
+             sh 'kubectl --token $api_token --server http://127.0.0.1:45263/  --insecure-skip-tls-verify=true apply -f deployment.yaml '
```
Comment (abaidgulshan) [inline Jenkinsfile]: Add server address variable as well 
k8s_server = "http://127.0.0.1:45263/"

**Review by abaidgulshan — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (yahyagulshan) [reply Jenkinsfile]: done

File: Jenkinsfile (hunk 1 above)
Comment (yahyagulshan) [reply Jenkinsfile]: done

**Review by abaidgulshan — APPROVED** [review]
LGTM!

**Pull request merged**